### PR TITLE
spdlog to system include and remove sinks when not needed

### PR DESCRIPTION
**Pull request by pablogh-2000**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,10 @@
 
 Setup | Status
 --- | ---
-Minimal | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-core%2Fmaster)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-core/job/master/)
-ZMQ | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-zmq-core%2Fmaster)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-zmq-core/job/master/)
-ROS 16.04 | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-ros-core-16.04%2Fmaster)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-ros-core-16.04/job/master/)
-ROS 18.04 | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-ros-core-18.04%2Fmaster)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-ros-core-18.04/job/master/)
+Minimal | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-core%2Fbugfix%252Fspdlog-include-refactor)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-core/job/bugfix%2Fspdlog-include-refactor/)
```
Comment (mharshe) [inline README.md]: These changes to Readme should not be included in the PR.

Comment (pablogh-2000) [reply README.md]: Yes I know. I am trying to figure out a way to do it differently to having a pre-commit hook. I will ensure to revert this back before merging
